test/unit/math/mix/fun/sd_test.cpp

```diff
@@ -6,13 +6,21 @@ void expect_sd(const T& m, const stan::test::ad_tolerances& tols
   auto f = [](const auto& x) { return stan::math::sd(x); };
   Eigen::VectorXd v(m.size());
   Eigen::RowVectorXd rv(m.size());
+  std::vector<double> sv(m.size());
   for (int i = 0; i < m.size(); ++i) {
     v(i) = m(i);
     rv(i) = m(i);
+    sv[i] = m(i);
   }
+
   stan::test::expect_ad(tols, f, v);
   stan::test::expect_ad(tols, f, rv);
   stan::test::expect_ad(tols, f, m);
+  stan::test::expect_ad(tols, f, sv);
+
+  stan::test::expect_ad_matvar(tols, f, v);
+  stan::test::expect_ad_matvar(tols, f, rv);
+  stan::test::expect_ad_matvar(tols, f, m);
```

Comment:
We will need this to work with `std::vector<var<mat>>` though. Haven't looked at it yet.